Getting Started in Angular 13 not working

I am trying to implement the getting started in an Angular 13 application. I followed the readme file, but I get the error:

Error: node_modules/@maxgraph/core/dist/types/gui/MaxPopupMenu.d.ts:29:15 - error TS2420: Class 'MaxPopupMenu' incorrectly implements interface 'Partial<PopupMenuItem>'.
  Types of property 'eventListeners' are incompatible.
    Type 'EventListenerObject[]' is not assignable to type '(eventName?: string) => EventListenerOrEventListenerObject[]'.
      Type 'EventListenerObject[]' provides no match for the signature '(eventName?: string): EventListenerOrEventListenerObject[]'.

29 declare class MaxPopupMenu extends EventSource implements Partial<PopupMenuItem> {

--- a/packages/core/src/gui/MaxPopupMenu.ts
+++ b/packages/core/src/gui/MaxPopupMenu.ts
@@ -51,7 +51,7 @@ import { PopupMenuItem } from '../types';
  *
  * Fires after the menu has been shown in <popup>.
  */
-class MaxPopupMenu extends EventSource implements Partial<PopupMenuItem> {
+class MaxPopupMenu extends EventSource {
   constructor(
     factoryMethod?: (handler: PopupMenuItem, cell: Cell | null, me: MouseEvent) => void
   ) {

